README.md

```diff
@@ -137,9 +137,42 @@ helper
 [See the JSDocs](https://fauna.github.com/faunadb-js/PageHelper.html) for
 more information on the pagination helper.
 
+#### Timeouts
+
+The client can be configured to handle timeouts in two different ways:
+
+1. Add a `timeout` field to the `options` block when instantiating the client
+2. By setting a `queryTimeout` on the client (or passing the value to the client's `.query()` method directly)
+
+The first option (i.e. `timeout`) represents a HTTP timeout on the client side. Represented in milliseconds, the client will wait the specified period before timing out if it has yet to receive data.
+
+```javascript
+const client = new faunadb.Client({
+  secret: 'YOUR_FAUNADB_SECRET',
+  timeout: 100,
+})
+```
+
+On the other hand, using the client's `queryTimeout` dictates how long FaunaDB will process the request on the server-side before timing out if it hasn't finished it's lookup and/or returned data. This can be done in two different ways:
```

Comment:
```suggestion
On the other hand, using the client's `queryTimeout` dictates how long FaunaDB will process the request on the server before timing out if it hasn't finished running the operation. This can be done in two different ways:
```